environment.yml

```diff
@@ -7,7 +7,7 @@ name: scipy-dev
 channels:
   - conda-forge
 dependencies:
-  - python
+  - python<=3.9
```

Comment:
I would not change this if it's only for the doc. This file is used for normal dev env creation.
```suggestion
  - python
```